config/clients/java/template/config-Configuration.java.mustache

```diff
@@ -0,0 +1,165 @@
+{{>licenseInfo}}
+package {{invokerPackage}};
+
+import static dev.openfga.util.StringUtil.isNullOrWhitespace;
+
+import dev.openfga.errors.FgaInvalidParameterException;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.net.http.HttpClient;
+import java.net.http.HttpConnectTimeoutException;
+import java.net.http.HttpRequest;
+import java.time.Duration;
+import java.util.regex.Pattern;
+
+/**
+ * Configurations for an ApiClient.
+ */
+public class Configuration implements BaseConfiguration {
+    public static final String VERSION = "0.1";
+
+    private static final String DEFAULT_USER_AGENT = "openfga-sdk java/0.0.1";
+    private static final Duration DEFAULT_READ_TIMEOUT = Duration.ofSeconds(10);
+    private static final Duration DEFAULT_CONNECT_TIMEOUT = Duration.ofSeconds(10);
+
+    private String apiUrl;
+    private String userAgent;
+    private Duration readTimeout;
+    private Duration connectTimeout;
+
+    public Configuration(String apiUrl) {
+        this.apiUrl = apiUrl;
+        this.userAgent = DEFAULT_USER_AGENT;
+        this.readTimeout = DEFAULT_READ_TIMEOUT;
+        this.connectTimeout = DEFAULT_CONNECT_TIMEOUT;
+    }
+
+    /**
+     * Assert that the configuration is valid.
+     */
+    @Override
+    public void assertValid() throws FgaInvalidParameterException {
+        if (this.apiUrl == null) {
+            throw new FgaInvalidParameterException(
+                    "apiUrl", "Configuration", new NullPointerException("API URL cannot be null."));
```

Comment:
```suggestion
                    "apiUrl", "Configuration", new NullPointerException("API URL cannot be null."));
```

Don't throw a null pointer exception here. And if apiUrl is null, let's default to `http://localhost:8080`

Reply:
Got it

Reply:
Added here: [55acd65](https://github.com/openfga/sdk-generator/pull/173/commits/55acd6518431a4a0c2c6fd2650c50a4ebccef206)